### DISCUSSION TITLE
Hetzner will be removing alternative cluster autoscaler images soon

### DISCUSSION THREAD

**Discussion by asemarian** [kube-hetzner/terraform-hcloud-kube-hetzner#1504] [Q&A] [answered]
I just got an email from Hetzner:

> If you were using the alternative container images we provided ([docker.io/hetznercloud/cluster-autoscaler](http://docker.io/hetznercloud/cluster-autoscaler)), please switch back to the images built by the Kubernetes community ([registry.k8s.io/autoscaling/cluster-autoscaler](http://registry.k8s.io/autoscaling/cluster-autoscaler)). We will remove the alternative container images on 19 November 2024.

So these alternative images will be removed soon. That means we need to basically undo [this](https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/pull/1487) change in time, right?

**Comment by BrammyS** [comment]
Made an issue for this #1505.

**Comment by BrammyS** [comment, marked as answer]
Also opened a PR for this #1506 :)
